src/input_buttons.h

```diff
@@ -66,16 +74,84 @@ namespace Input {
 		BUTTON_COUNT
 	};
 
-	/**
-	 * Initializes input buttons to their mappings.
-	 */
-	void InitButtons();
-
-	/** Buttons list of equivalent keys. */
-	extern std::vector<std::vector<int> > buttons;
+	constexpr auto kButtonNames = makeEnumTags<InputButton>(
+		"UP",
+		"DOWN",
+		"LEFT",
+		"RIGHT",
+		"DECISION",
+		"CANCEL",
+		"SHIFT",
+		"N0",
+		"N1",
+		"N2",
+		"N3",
+		"N4",
+		"N5",
+		"N6",
+		"N7",
+		"N8",
+		"N9",
+		"PLUS",
+		"MINUS",
+		"MULTIPLY",
+		"DIVIDE",
+		"PERIOD",
+		"DEBUG_MENU",
+		"DEBUG_THROUGH",
+		"DEBUG_SAVE",
+		"TOGGLE_FPS",
+		"TAKE_SCREENSHOT",
+		"SHOW_LOG",
+		"RESET",
+		"PAGE_UP",
+		"PAGE_DOWN",
+		"SCROLL_UP",
+		"SCROLL_DOWN",
+		"FAST_FORWARD",
+		"TOGGLE_FULLSCREEN",
+		"TOGGLE_ZOOM",
+		"BUTTON_COUNT");
 
-	/** Direction buttons list of equivalent buttons. */
-	extern std::vector<std::vector<int> > dir_buttons;
+	constexpr auto kButtonHelp = makeEnumTags<InputButton>(
+		"Up Direction",
+		"Down Direction",
+		"Left Direction",
+		"Right Direction",
+		"Decision Key",
+		"Cancel Key",
+		"Shift Key",
+		"Number 0",
+		"Number 1",
+		"Number 2",
+		"Number 3",
+		"Number 4",
+		"Number 5",
+		"Number 6",
+		"Number 7",
+		"Number 8",
+		"Number 9",
+		"Plus Key",
+		"Minus Key",
+		"Multiply Key",
+		"Divide Key",
+		"Period Key",
+		"(Test Play) Calls the Debug Menu",
+		"(Test Play) Walk through walls",
+		"(Test Play) Call the save menu",
+		"Call the settings menu",
```

Comment:
Because this is visible to users better use "Open" instead of "Call".

Reply:
Fixed